python/lsst/sconsUtils/utils.py

```diff
@@ -241,3 +241,19 @@ def nested(member):
             setattr(scope, kw["name"], member)
         return member
     return nested
+
+
+def get_conda_prefix():
+    """Returns a copy of the current conda prefix."""
+    if os.environ.get('CONDA_BUILD', "0") == "1":
+        # when running conda-build, the right prefix to use is PREFIX
+        # however, this appears to not be around in some builds, so I
+        # am going to default back to CONDA_PREFIX
+        if 'PREFIX' in os.environ:
+            _conda_prefix = os.environ['PREFIX']
+        else:
+            _conda_prefix = os.environ['CONDA_PREFIX']
```

Comment:
I don't like this block, but it fixed a build problem. Conda build has three envs defined: the build env, the host env, and the run env. Usually, the compilers live in the build env, which is the one that gets activated at CONDA_PREFIX IIUIC. The host env is in PREFIX. When building the stack with conda build, I have been using the option to merge the host and build envs. This makes eups happy and makes the build easier. I had expected to still have PREFIX around during this and it appears to be so in the CI shell. However, once sconsUtils gets to this part, PREFIX is no longer in the env. IDK why, but CONDA_PREFIX will work. I have left the proper one in just in case something changes later.